packages/graphql-key-transformer/package.json

```diff
@@ -27,7 +27,8 @@
     "graphql-dynamodb-transformer": "6.19.14",
     "graphql-mapping-template": "4.15.2",
     "graphql-transformer-common": "4.17.10",
-    "graphql-transformer-core": "6.21.5"
+    "graphql-transformer-core": "6.21.5",
+    "ts-md5": "^1.2.7"
```

Comment:
please use "md5" that we are already using in transformer.

Reply:
done!!